### DISCUSSION TITLE
授权码过期，请大佬帮忙激活一下 感谢

### DISCUSSION THREAD

**Discussion by zhoubei9** [LFTEC/LearningFucker#53] [General]
请将下面代码复制到Github讨论区中，收到激活码后，调用activate命令进行激活
------------------------------License------------------------------
U8cyh35jh09xG9LJe+kYJD5uVHc2ZBgFeieAhvJkTw6a3avNUnkBYbUi5J5imIxV2K0
mpG1kB1InrD0USDljgAnK+P7ipipBBeOGtjEcdYF6i16TCId6xbLpaFw3wq8qQPx1Tf
n0WO/N/9NSu77rXoOXmUZ7L7WuGGYn8nxDPXF8VJrxANleYSF7xcZpSRbvomsUbtn7F
QPT0Pgawo2Uoe8dNzasPbj8BlcCPCrJx+Zc484vY7E56lwGxMQpYJqOQciurcwkGc3x
Shtgyw0EamMlGi1LkV4pTIJJ3ZHJ+FihMCX0TCtT32NPmo5wkj7ABkw/mh9lFjUTIyN
d7R2FbVjaBX2pHMh0Bfa0yqJA/BcHyZNjGSeeBWeCxPj5W1BDvziu/WwX/7Jlbip+uV
idb8WCruGsG5h8wC4UEDZvYXFRreb8iNWEVj/Uml3DnCVv4aZy+uDWeKWLDV0+SwCXP
J95mEOK641lCbTmWyYQkwPCQaY+8VW2Pa1kLtImn4PjwihDkZluMrRWOxdGpGLCG/6o
XtkA4pqcdWHR8dpphmSdZQ0xtlHBrx292iAg58d/8qjQ3F29pUhnaxTFA4iwGAybx0T
H+mEn0ooBO+dfHIaacbhQfBOh+0kXNkHrCCPqz91iZHXQYlQxEDPTwRInOCyUPj3hDa
rf4ECeTX49TlU3VqWUIx+BKfVdgKlGQmkSM2KVB+Bw8iAr34EMUCV9twRCMDbwDVGMF
sFTOx7p+DQfnU1EOdzm9sQVm9V5MgRzTq8OgCHGCl+QzEaCj6vNbtWJn7rPbtUCl/HI
dViOV3mJj4Li5r49Uc3lzsPofABrV0yMymypegqymgCYGoybsmwwI/GVa3qUMkH0JfO
tomKxDyut1dvrEtD9qlzoGs9un4e1MfI+QBIm2MQGBwy+M1ohi40NPbqUaOQxnnzIrR
DfZNbUnH7/Xko3kQJH/P719vwnNlQv9VAjVztpEz+D/iD79Qiit+ylXHexUuweet1UU
JBhUOrGkeEIfBr4QBmcTgh4REg3GVZb1WC28zxAbKScEpl3r8E55AFiXTWr5rW/LmjU
lYqOX4XHIhjG4QCvTRwDtFserhF10/8NyoUYv8L73dTPaK9mEdvQq48y64G3SZM5Ues
PvEK7xbFLd76wvovTWWQb/DLtfJRxy3cVpx7JbESQK7rWzVMiB3WRUP9smyXDMM9PzG
dnwjwUc2V3IiEEqJSn0x4JuZEwItNtqLNyldgBN5rB7yVdwf6MNIq4FWaGp6L5VBXDm
31ZfoShTlJQsS0F5i6ofG6dIZdjvHwgaqwbvhTZ0uNpZCsuPyls5tqZIxHj0XwN6lz7
D07xuAu02d+3/fuNWV3+uHMncg==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
PIDTU/KQAJdRGbNuzubxDHnZo7tOUEx+vLh0CSI2Q0YvJ2rzQRQbdds5tNyYlDXnzesEwAzDwDI9MEcZLmKA4GKHnN9S32d6tVSfJSXUCL7E534HWPcJdizW8QnoyM4c+VCcHKd1RttI4tynONXhFgEE9ikYV1Uj/cvyOmKCpvtvQvNccqHtrOWmsJ6ka+c4q3RvFmbWfqDRk5VvljiP5k5QbeXgb21gELP5OSMrIYqayM07SRYfHdrb3cbBEpNIZ9IGi+Y94EL7C6Ogr8GHACd8DWnnnAvXQQQAbnG9oSiKaEKMaku4LtRZclNooung9YbqfFSlsRrtgNVtwMXuxgfv+TZJK8dbSRD3bVw2MVgekmjF6CRV7hbHGTpupzXcptKnJbBMAls7JW9d8flV/JBoWHJtnAf61wuExV8b0F2k8rletFCkl5YGmSd33ek9140QkSKy6VGDXqSE1s5bkcaKZfJgmmyX6jQG91amRRFhL3YhdngkkwXaNLsrjUiURpbdWXMLuFEdfftxdAo2kMKrAB86VE4xeiNMO7B/i57Jj11IhXIIwlpqLR+NKLPx0+DlYmvE8ZFSiKFqkce/jZBa+TQDTdhOUqiOgXJ6J8RkjkpO12f37sAki2XbiCFLvbDXCyFkwpv9KAecRevFmLf73wL87KnSMH6yIgPn4V8aMJqXj633VAFyP7hxS+LZ4c0In/rhFU1PvKEpo/UZ5+FtC+D7qQK149GSxiAQZE0dkTqzqPNhCazFSHgkBlF3he9NtstopqHP1dcMjlCTB81uyYZQ5Xa2F8l5b8ijPklTYUoZY4aKqIITvRmqkf0iJVBCTd67SQXfUY+jnvVzpNsa+bnvIdYrKzLG507B20tO4I3KMOGCVaRFpJQc6on9BpTeer++eZdr4VSAMLk0ofB60FqxoY4ADa+PNIcgDpfn3H5KGQmmIZfkd3/LIdVUJsdANhWSRQr0MSDnSAqur0odDXDMy5pjRG4/wURfFAj9hdF08IFZx7KnDM1VKDKxCvHjuDpJzfg9BWbdLRQ/CmR6iUPLoOnQQRvaP7KGdSpW+RwXn8ts2ogHlMmuWvAO5e8LEdETBjL1TioXmiidlS7SpRCNOuvPplGWPUJz4e4wxHgL4H+JqufoSAKSnJmrsT+ufT9G4Ge4Zxw7/+hP2NI5avgvJU70nlBfJ/v1WADzY1hgq82EYpD7U2zds8AUe7RsOKwzYPzKSqAfHxjeYcR52qOIMO+s+ctYeQ8IKDi5eQ54CA8dAirklxn99YRNesIvCpRzqg+gci7tB5+FKEhSnPE+BfRHmrnSRzJiDg5grRavVQhzXrJ5qnOUlCPHrBxiKP78dTrbNCb/ZVYnxQ==
